### PR TITLE
change 3rd to 2nd person, fix typos, change mobilesync references to rse+

### PR DESCRIPTION
**Pull request by PaigeWarmker**
Makes consistent use of "you" rather than "they" or "users", per the style guidelines. Also fixes a few other typos, and removes "Mobile Sync" references in favor of "Rackspace Email Plus" (since Mobile Sync is no longer offered as a standalone service, but instead is bundled into the Rackspace Email Plus product).

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-35b962d5ac/how-to/). :bow:

**Comment by ReneeRendon** [comment]
Created a task in JIRA Info-dev 708

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-eae30ba60a/how-to/)

**Comment by stephamon** [comment]
Hey @catlook here are my suggestions. Let me know if you have any questions!

[Rackspace.Webmail.overview_SFedit.docx](https://github.com/rackerlabs/rackspace-how-to/files/1548325/Rackspace.Webmail.overview_SFedit.docx)
[Rackspace.Webmail.overview_SFedit.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1548327/Rackspace.Webmail.overview_SFedit.pdf)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4236f9ae1b/how-to/).

File: content/rackspace-email/rackspace-webmail.md [hunk 1]
```diff
@@ -1,102 +1,115 @@
...
 product: Rackspace Email
 product_url: rackspace-email
 ---
 
-Rackspace Email allows you to check and manage their email online
-through a Webmail interface. Webmail is a direct connection to your email without the need of additional software. Messages that are sent and received using Webmail will be waiting for you the next time you log into the portal. The Webmail portal provides most of the same functionality a mail client would offer, from sending and receiving email to setting up email signatures and even setting up email forwarding to your personal email accounts.
+With Rackspace Email, you can check and manage your email online through a webmail interface. Webmail provides direct access to your email without the need for additional software. Messages that are sent and received using webmail wait on the portal until the next time you log in. The webmail portal provides most of the same functionality that a mail client offers, including sending and receiving email, signatures, and email forwarding.
 
-### Log in to Webmail
+This article provides and overview of the available features in the Rackspace Email webmail interface.
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: an overview

File: content/rackspace-email/rackspace-webmail.md [hunk 2]
```diff
@@ -1,102 +1,115 @@
...
+### Log in to webmail
 
-When you log in, Rackspace allows several options:
+To log in to webmail, go to <https://apps.rackspace.com/>.
 
-- **Remember this info** - Allow Rackspace to interact with your browser to remember your login information
-- **Use SSL** - SSL stands for Secure Socket Layer, which means when you log in, your data is encrypted
+When you log in, Rackspace provides the following options:
+
+- **Remember this info** - Select teh checkbox to allow Rackspace to interact with your browser to remember your login information
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: the checkbox..... information. (period at end)

File: content/rackspace-email/rackspace-webmail.md [hunk 3]
```diff
@@ -1,102 +1,115 @@
...
 
 ### Calendar
 
-Rackspace Email allows you the ability to manage their appointments and meetings through our Webmail interface. Users
-have the ability to share their calendar, create personal calendars, import events and add shared calendars within your domain. Rackspace also allows you to sync your calender events with your mobile device as well with Rackspace Mobile Sync.
+Rackspace Email enables you to manage your appointments and meetings through our webmail interface. You can share your calendar, create personal calendars, import events and add shared calendars within your domain. You can also sync your calendar events with your mobile device as well with Rackspace Email Plus. For more information about Rackspace Email Plus, see [Rackspace Email Plus featuring Cloud Drive](https://www.rackspace.com/en-us/email- hosting/webmail/cloud-drive).
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: import events, and add (add comma)

your mobile device, as well as with Rackspace....

File: content/rackspace-email/rackspace-webmail.md [hunk 4]
```diff
@@ -1,102 +1,115 @@
...
 
 ### Tasks
 
-Rackspace Email allows users the ability to manage their tasks through our Webmail interface. Users have the ability to create
-new tasks, manage their tasks, and create a task list.
+Rackspace Email provides the ability to create and manage tasks and create task lists through our Webmail interface.
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: tasks, and create task lists...

File: content/rackspace-email/rackspace-webmail.md [hunk 5]
```diff
@@ -1,102 +1,115 @@
...
-Rackspace Email provides various options for users to manage their email account. While in the Webmail interface, click the menu beside your email address in the top-right corner and select **Settings** to see a list of features and options available for users.
+Rackspace Email provides various options for you to manage your email account. While in the webmail interface, click the menu beside your email address in the top-right corner and select **Settings** to see a list of features and options available.
 
-#### General Settings: Email options
+#### General Settings: 
 
-- **Display Preferences** provides several options for users to display HTML emails, enable shortcuts, changing your viewing pane and change the number of messages displayed in your reading pane.
-- **New Messages** provides the option to play an alert for new the arrival of new messages and the option to choose how often to check for new messages.
-- **Trash Options** provides the option to move deleted email to the trash or immediately purge upon deletion.
+In the General Settings section, you can manage settings for email options, your calendar, and your preferred language and date and time settings.
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: calendar, your preferred language, and date and time settings.

File: content/rackspace-email/rackspace-webmail.md [hunk 6]
```diff
@@ -1,102 +1,115 @@
...
 
 <img src="{% asset_path rackspace-email/rackspace-webmail/Webmail.png %}" alt="" />
 
-#### General Settings: Calendar
+##### Email options
+
+- **Display Preferences** provides several options for you to display HTML emails, enable shortcuts, change your viewing pane, and change the number of messages displayed in your reading pane.
+- **New Messages** provides the option to play an alert for the arrival of new messages and the option to choose how often to check for new messages.
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: new messages, and the option

File: content/rackspace-email/rackspace-webmail.md [hunk 7]
```diff
@@ -1,102 +1,115 @@
...
+- **Replying & Forwarding Citations** selects whether you want the original composed message to be included in your reply and to set a user defined start text and end text of your choice.
+
+##### Identities
 
 - **Add New Identity** creates a new identity, which enables you to quickly change the name, email address, and reply address on your outgoing email.
 
-#### Composing Email: Signatures
+##### Signatures
 
 - **Add New Signature** adds a new signature and assign it to your outgoing email and also to specific identities. To learn more, see [Add a Rackspace Email signature](/how-to/adding-a-signature-to-rackspace-email).
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: and assigns

File: content/rackspace-email/rackspace-webmail.md [hunk 8]
```diff
@@ -1,102 +1,115 @@
...
+
 - **Auto-Reply** activates the auto-reply feature for times when you are out of the office.
 - **Forwarding** forwards any mail to any email address with the option to save a copy in your inbox.
 - **Filtering** provides the ability to create filters for specific incoming email and to route them to a specified folder.
 
-<img src="{% asset_path rackspace-email/rackspace-webmail/Webmail3.png %}" alt="" />
+#### Spam Settings
 
-#### Spam Settings: Preferences
+In the Spam Settings section, you can manage settings for spam preferences, and safelists and blacklists.
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-webmail.md]: spam preferences, safelists, and blacklists.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a525bfb20c/how-to/)

**Pull request merged**